### ISSUE TITLE
Trainig Issue: Obj: 0.000000, No Obj: 0.000000

### ISSUE THREAD

**Issue by manjWu** [pjreddie/darknet#1239]
Hi, I'm going to achieve resnet101_HR(Finding Tiny faces), which is based on the resnet101 and have good performance for small objects detection.  And the cfg file is shown below.

`[net]
# Training
batch=16
subdivisions=2

# Testing
# batch=1
# subdivisions=1

height=256
width=256
channels=3
min_crop=128
max_crop=448

burn_in=1000
learning_rate=0.0001
policy=poly
power=4
max_batches=800000
momentum=0.9
decay=0.0005

angle=7
hue=.1
saturation=.75
exposure=.75
aspect=.75
......

# score_res4
[convolutional]
filters=126 # (class+coords+1)*num
size=1
stride=1
pad=1
activation=linear

# score4
[upsample]
stride=2

# conv4之前
[route]
layers=-95 

# score_res3
[convolutional]
filters=126 # (class+coords+1)*num=(2+4+1)*18
size=1
stride=1
pad=1
activation=linear

#-------------------------------------
[shortcut]
from=-3
activation=linear

[region]
anchors = 8.0413,7.8414, 18.1484,16.8480, 33.7102,17.9955, 29.5356,35.1608, 53.5160,32.6654, 47.1668,60.5287, 83.3596,45.0115, 83.0197,100.4758, 133.5783,77.9650, 76.1687,183.0120, 131.0187,120.0259, 197.6056,94.5544, 184.0026,174.3252, 229.9368,146.4720, 165.9355,205.5448, 148.0733,251.2000, 227.9670,200.6440, 239.0800,247.4151
bias_match=1
classes=2
coords=4
num=18 # 每个grid预测的BoundingBox个数 
softmax=1
jitter=.3
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .6
random=1
`

 But when training, the Obj and No Obj are always zeros. 

` 3123: 0.355213, 0.563204 avg loss, 0.000197 rate, 0.567173 seconds, 99936 images
Loaded: 0.000078 seconds
Region Avg IOU: 0.486892, Class: 0.827141, Obj: 0.000000, No Obj: 0.000000, Avg Recall: 0.600000,  count: 5
Region Avg IOU: 0.421099, Class: 0.487379, Obj: 0.000000, No Obj: 0.000000, Avg Recall: 0.333333,  count: 3
Region Avg IOU: 0.567073, Class: 0.766520, Obj: 0.000000, No Obj: 0.000000, Avg Recall: 0.600000,  count: 5
Region Avg IOU: 0.265414, Class: 0.669996, Obj: 0.000000, No Obj: 0.000000, Avg Recall: 0.000000,  count: 4`

Could you help me?

Thank you
